package.json

```diff
@@ -7,10 +7,12 @@
     "dev:client": "NODE_ENV=development webpack -w",
     "dev:server": "NODE_ENV=development nodemon -w dist/client -w src/server/ src/server/index.js",
     "prod:build": "NODE_ENV=production webpack -p",
-    "format": "prettier --write 'src/**/*.js'",
+    "format": "prettier --write '{src,test}/**/*.js'",
     "lint": "eslint --cache --max-warnings 0",
-    "lint:all": "npm run lint -- 'src/**/*.js'",
-    "test": "npm run lint:all",
+    "lint:all": "npm run lint -- '{src,test}/**/*.js'",
+    "pretest": "npm run lint:all",
+    "test": "mocha '{src,test}/**/*.test.js'",
+    "test:watch": "nodemon -w src -w test -x 'npm test'",
     "precommit": "lint-staged"
   },
   "lint-staged": {
```

Comment:
Do you think we should run the unit tests on precommit? We are doing it in popups and it hasn't been a problem since the unit tests have remained quite snappy. I'm fine just leaving `npm test` there.